src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -73,6 +74,7 @@ class DropDownGroup extends React.Component {
         e.preventDefault();
         this.openDropdown();
         break;
+      case ENTER:
```

Comment:
Is this a global listener? I am afraid that this could conflict with a search input's `onKeyDown` listener 🤔